code/datums/mood_events/generic_negative_events.dm

```diff
@@ -335,11 +335,6 @@
 	mood_change = -2
 	timeout = 5 MINUTES
 
-/datum/mood_event/tail_regained_wrong
```

Comment:
why did you remove this event is it unused?

Reply:
`del: removes negative moodevent on getting wrong species tail`

`I also removed the mood-event on getting the wrong tail. It was turbo shitcode and causing so many hard-dels for someting that discourages frankensteining???`

Reply:
add it back

Reply:
@Time-Green ^

Reply:
AAAARGH